determined-ai/determined
# chore: Return trial workloads and metrics in place of ExperimentRaw [DET-6485]

## Description

The CLI `experiment describe` responds ends with a table of steps with an experiment's checkpoint and validation information. If the user adds the `--metrics` flag we should see more columns with any training and validation metrics.

This helps us remove the old GetExperiment and related ExperimentRaw, ExperimentListRaw etc code

## Test Plan

**Updated 3/15** The majority of changes (by lines) are to e2e_tests.  When testing this PR, focus should be on the `det experiment` CLI and outputs , which should look the same as the current CLI.

- config
- create [should create an experiment and follow the log output]
- describe [including `--metrics`]
- download-model-def
- list-checkpoints [including `--best`, `sortBy=SORT_BY_SEARCHER_METRIC`]
- list-trials

We should also ensure that no other parts of the code call the old API endpoints for: `/:experiment_id` (+ /checkpoints, /config, /model_def, /summary, /metrics/summary).  For example we found that `/experiments/:experiment_id/preview_gc` is still call-able by the API, so it should stay in.

## Checklist
